subsystem/elevator.py

```diff
@@ -0,0 +1,52 @@
+import rev
+from robotpy_toolkit_7407 import Subsystem
+from robotpy_toolkit_7407.motors.rev_motors import SparkMax, SparkMaxConfig
+from robotpy_toolkit_7407.sensors.limit_switches import MagneticLimitSwitch
+
+import config
+import constants
+from units.SI import meters
+
+# TODO: Change config once robot is built
+ELEVATOR_CONFIG = SparkMaxConfig(
+    1.5, 0, 0.004, 0.00017, (-1, 1), idle_mode=rev.CANSparkMax.IdleMode.kBrake
+)
+
+
+class Elevator(Subsystem):
+
+    motor_extend: SparkMax = SparkMax(
+        config.elevator_can_id, config=ELEVATOR_CONFIG, inverted=True
+    )
+
+    def __init__(self) -> None:
+        super().__init__()
+
+        self.elevator_bottom_sensor = None
+
+    def init(self) -> None:
+        self.motor_extend.init()
+        self.elevator_bottom_sensor = MagneticLimitSwitch(config.magnetic_limit_switch_port)
```

Comment:
For the magnetic limit switch, we will be attaching it to the spark max motor controller. I would recommend looking at some of the older code for the arm, as that has the correct function for that